etl_manager/meta.py

```diff
@@ -368,8 +338,15 @@ def s3_database_path(self) :
         Returns the s3 path to the database
         """
         return "s3://{}/{}{}".format(self.bucket, self.s3_base_folder, self.location)
```

Comment:
Just thinking that we are building in inconsistency around having properties with '/' and no '/' we should pick one. I think we went with no '/' for this package? Should we quickly run through stuff and make it consistent in this PR?